### PR TITLE
PBjs Core : User sync iframe over image

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change
- [ X ] Bugfix

## Description of change
Fix for https://github.com/prebid/Prebid.js/issues/7330

### CODE REVIEW

**Comment by pm-harshad-mane** [comment]
Hello @Rothalack 
Could you please review the fix?

**Comment by Rothalack** [comment]
This looks good to me. The history of merges in this branch should be fine. For a clean branch in the future you may want to do a rebase @pm-harshad-mane , but again, should be fine for this PR.

Am I considered the second review on this since you looked at it? @ChrisHuie Let me know if you want another reviewer or if I'm clear to merge this.

**Review by Rothalack — APPROVED** [review]

**Comment by ChrisHuie** [comment]
You are good to merge this. Thanks @Rothalack and @pm-harshad-mane

**Pull request merged**
